src/main/java/io/strimzi/kafka/metrics/PrometheusMetricsReporterConfig.java

```diff
@@ -64,11 +75,15 @@ public class PrometheusMetricsReporterConfig extends AbstractConfig {
     private static final ConfigDef CONFIG_DEF = new ConfigDef()
             .define(LISTENER_CONFIG, ConfigDef.Type.STRING, LISTENER_CONFIG_DEFAULT, new ListenerValidator(), ConfigDef.Importance.HIGH, LISTENER_CONFIG_DOC)
             .define(ALLOWLIST_CONFIG, ConfigDef.Type.LIST, ALLOWLIST_CONFIG_DEFAULT, ConfigDef.Importance.HIGH, ALLOWLIST_CONFIG_DOC)
-            .define(LISTENER_ENABLE_CONFIG, ConfigDef.Type.BOOLEAN, LISTENER_ENABLE_CONFIG_DEFAULT, ConfigDef.Importance.HIGH, LISTENER_ENABLE_CONFIG_DOC);
+            .define(LISTENER_ENABLE_CONFIG, ConfigDef.Type.BOOLEAN, LISTENER_ENABLE_CONFIG_DEFAULT, ConfigDef.Importance.HIGH, LISTENER_ENABLE_CONFIG_DOC)
+            .define(METRIC_NAME_PREFIX_CONFIG, ConfigDef.Type.STRING, METRIC_NAME_PREFIX_DEFAULT, ConfigDef.Importance.HIGH, METRIC_NAME_PREFIX_DOC);
```

Comment:
We don't want this to be a configuration. The value comes from `KafkaPrometheusMetricsReporter`.